docs/rfc/rfc-027-p2p-message-bandwidth-report.md

```diff
@@ -0,0 +1,137 @@
+# RFC 27: P2P Message Bandwidth Report
+
+## Changelog
+
+- Nov 7, 2022: initial draft (@williambanfield)
+- Nov 15, 2022: draft completed (@williambanfield)
+
+## Abstract
+
+Node operators and application developers complain that Tendermint nodes consume
+larges amounts of network bandwidth. This RFC catalogues the major sources of bandwidth
+consumption within Tendermint and suggests modifications to Tendermint that may reduce
+bandwidth consumption for nodes.
+
+## Background
+
+Multiple teams running validators in production report that the validator consumes a lot of bandwidth.
```

Comment:
Just to confirm, is this data coming from a validator node? I assume this node are private and should pretty few peers.

Reply:
It appears that this was a validator by the presence of the `validator_last_signed_height` metric. 
![image](https://user-images.githubusercontent.com/4561443/203173724-5f639de7-7062-4ddb-9219-46a3f198360b.png)